pkg/config/config.go

```diff
@@ -188,10 +206,11 @@ func NewMicroshiftConfig() *MicroshiftConfig {
 	}
 
 	return &MicroshiftConfig{
-		LogVLevel:       0,
+		LogVLevel:       2,
```

Comment:
I think this is because the new default LogLevel is "Normal" which is equivalent to "2" not "0". This is the standard verbosity setting that is currently used on [KAS operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L492) as well.

Reply:
Correct. It also fixes a problem that without `config.yaml` present, we've been defaulting to 0 and with `config.yaml` present, to 2 (https://github.com/openshift/microshift/blob/main/pkg/config/config.go#L329 and https://github.com/openshift/microshift/blob/main/pkg/config/config.go#L533-L534).

Reply:
OK